debian-snap-base/Dockerfile.tmpl

```diff
@@ -44,7 +44,7 @@ RUN set -x && mkdir -p /var/run/s6 && \
     # Configure Chrome repo.
     curl -s https://dl-ssl.google.com/linux/linux_signing_key.pub -o linux_signing_key.pub && \
     apt-key add linux_signing_key.pub && \
-    echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list && \
+    echo "deb [arch=${TARGETARCH}] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list && \
```

Comment:
Nice try, but there is no chrome stable for arm.

Reply:
Well, that image wont get used then. :\